Fix mobile layout

antstorm opened the pull request:
Here's a fix for mobile layout as promised. I've made it more compact, here's how it looks on the iPhone 6:

![](https://photos-3.dropbox.com/t/2/AADGv1g8DKRk13PxSXbQM0kBcLsGSRvX0-CSSjTeu1oAHg/12/7209271/png/1024x768/3/1428444000/0/2/Screenshot%202015-04-07%2021.51.49.png/CLeCuAMgASACIAMoASgC/8XbUuVocTs2PHymNlorSV9_kz03ON0N_anFisdbO4S4)

Single line posts are still the height of the source icon, while longer titles take a bit more space.

goddamnyouryan commented:
Awesome! Thanks so much!

antstorm commented:
@goddamnyouryan have you deployed these last 2 PRs? Somehow I don't see any changes…

goddamnyouryan commented:
@antstorm yep i deployed both to the web app and the chrome store. Maybe try restarting your chrome? Otherwise can you send me a screenshot of it not working?

antstorm commented:
@goddamnyouryan I've just opened CSS file directly (http://www.coven.link/stylesheets/application.css) and tried refreshing it/incognito/different browser and it seems like it does not include the latest fixes.

goddamnyouryan commented:
@antstorm very strange, they are definitely there for me:

![screen shot 2015-04-08 at 10 40 35 am](https://cloud.githubusercontent.com/assets/377331/7051249/c1dfd2d2-dddb-11e4-9885-83fa21af7468.png)

I've gone ahead and invalidated the files on cloudfront again, hopefully they will work for you in the next 15 minutes

antstorm commented:
@goddamnyouryan this is the locally built application.css that I've got:

```css
html,body,div,span,applet,object,iframe,h1,h2,h3,h4,h5,h6,p,blockquote,pre,a,abbr,acronym,address,big,cite,code,del,dfn,em,img,ins,kbd,q,s,samp,small,strike,strong,sub,sup,tt,var,b,u,i,center,dl,dt,dd,ol,ul,li,fieldset,form,label,legend,table,caption,tbody,tfoot,thead,tr,th,td,article,aside,canvas,details,embed,figure,figcaption,footer,header,hgroup,menu,nav,output,ruby,section,summary,time,mark,audio,video{margin:0;padding:0;border:0;font:inherit;font-size:100%;vertical-align:baseline}html{line-height:1}ol,ul{list-style:none}table{border-collapse:collapse;border-spacing:0}caption,th,td{text-align:left;font-weight:normal;vertical-align:middle}q,blockquote{quotes:none}q:before,q:after,blockquote:before,blockquote:after{content:"";content:none}a img{border:none}article,aside,details,figcaption,figure,footer,header,hgroup,main,menu,nav,section,summary{display:block}@font-face{font-family:"fontawesome-webfont";src:url("/fonts/fontawesome-webfont.eot");src:url("/fonts/fontawesome-webfont.eot?#iefix") format("embedded-opentype"),url("/fonts/fontawesome-webfont.woff") format("woff"),url("/fonts/fontawesome-webfont.ttf") format("truetype"),url("/fonts/fontawesome-webfont.svg#fontawesome-webfont") format("svg");font-weight:normal;font-style:normal}html,html a{-webkit-font-smoothing:antialiased;text-shadow:1px 1px 1px rgba(0,0,0,0.004);line-height:1.5;letter-spacing:-0.04em}html,body,#container{height:100%}*{box-sizing:border-box}body{font-family:"Helvetica Neue", "Helvetica", Arial, sans-serif;font-size:1rem;background:#ECF0F1;color:#3f4750}a{-moz-transition:color 0.2s ease-in-out;-o-transition:color 0.2s ease-in-out;-webkit-transition:color 0.2s ease-in-out;transition:color 0.2s ease-in-out;color:#3f4750}a:hover{color:#EB974E}header{overflow:hidden;*zoom:1;background:white;color:#3f4750;padding:20px 30px}header a{color:#3f4750}header h1{float:left}header h1 a{background:url("/images/logo.png") no-repeat;height:22px;width:100px;display:block;text-indent:-9999px}header .nav-toggle{float:right;font-family:"fontawesome-webfont", "Helvetica", Arial, sans-serif;font-size:1rem;cursor:pointer}header .nav-toggle:before{content:'\f0c9'}header{overflow:hidden;*zoom:1;background:white;color:#3f4750;padding:20px 30px;position:fixed;z-index:3;width:100%;height:60px;border-bottom:1px solid #ECF0F1}header a{color:#3f4750}header h1{float:left}header .menu-toggle{float:right}nav{background:#a4adb7;color:#ECF0F1}#container{-moz-transition:-moz-transform 0.2s ease-in-out;-o-transition:-o-transform 0.2s ease-in-out;-webkit-transition:-webkit-transform 0.2s ease-in-out;transition:transform 0.2s ease-in-out;background:#ECF0F1 url("/images/preloader.gif") no-repeat center center;position:relative;z-index:2;width:100%;padding:60px 30px 0}#container.menu-open{-moz-transform:translate(-300px, 0);-ms-transform:translate(-300px, 0);-webkit-transform:translate(-300px, 0);transform:translate(-300px, 0)}#container.loaded{min-height:100%;height:auto;background-image:none}footer a{color:#ECF0F1}footer a:hover{color:#3f4750}@media screen and (min-width: 320px) and (max-width: 640px){#container{padding:60px 20px 0}}nav{background:#a4adb7;color:#ECF0F1;z-index:1;width:300px;position:fixed;right:0;top:0;height:100%;padding:80px 20px 20px}ul.sources{margin:25px 0}ul.sources li{display:inline-block;width:45%;margin:0 5% 5% 0;background:#8794a1;cursor:pointer}ul.sources li:nth-of-type(2n){margin:0 0 5% 0}ul.sources li input[type=checkbox]{display:none}ul.sources li input[type=checkbox]:checked+label{outline:4px solid #ECF0F1}ul.sources li label{display:block;text-align:center;font-size:0.85rem}ul.sources li label:before{height:100px;color:white;display:block;font-size:4rem}ul.buttons{font-family:"Helvetica Neue", "Helvetica", Arial, sans-serif;font-size:0.85rem;font-weight:bold}ul.buttons li{margin-bottom:15px}ul.buttons a{-moz-border-radius:3px;-webkit-border-radius:3px;border-radius:3px;background:#ECF0F1;display:block;text-align:center;text-decoration:none;padding:10px;color:#6c7a89}ul.buttons a:hover{-moz-transition:all 0.2s ease-in-out;-o-transition:all 0.2s ease-in-out;-webkit-transition:all 0.2s ease-in-out;transition:all 0.2s ease-in-out;color:#3f4750;background:#cfd9db}ul.posts{padding:50px 0}ul.posts li{overflow:hidden;*zoom:1;margin:0 0 1.5rem 0;background:white;height:60px;padding-right:10px}ul.posts li:before{color:white;font-size:2.5em;text-align:center;width:60px;height:60px;display:block;position:absolute}ul.posts .info{float:left;margin-left:80px}ul.posts h2{font-size:1.5rem}ul.posts h2 a{display:inline-block;line-height:60px}ul.posts h2 a:visited{color:#a4adb7}ul.posts date{font-size:0.75rem;color:#6c7a89}ul.posts h3{float:right;position:relative;width:40px}ul.posts h3 a{position:relative;z-index:2;display:block;text-align:center;margin:18px 0 0 -1px;font-weight:bold;color:white}ul.posts h3:after{font-family:"fontawesome-webfont", "Helvetica", Arial, sans-serif;font-size:2.5rem;-moz-transition:color 0.2s ease-in-out;-o-transition:color 0.2s ease-in-out;-webkit-transition:color 0.2s ease-in-out;transition:color 0.2s ease-in-out;content:'\f075';color:#cfd9db;position:absolute;top:0;left:0;z-index:1}ul.posts h3:hover:after{color:#EB974E}ul.posts a{text-decoration:none}@media screen and (min-width: 320px) and (max-width: 640px){ul.posts{padding:20px 0}ul.posts li{overflow:auto;height:auto;margin-bottom:20px}ul.posts li .info{margin:6px 0 6px 75px}ul.posts h3.comments{display:none}ul.posts h2{font-size:1.2rem}ul.posts h2 a{display:block;line-height:1.3;margin-bottom:6px}ul.posts h2 date{display:block}}.hacker-news:before{content:'Y';background:#EB974E}.lobsters:before{content:'L';background:#96281B}.product-hunt:before{content:'P';background:#DA552F}.r-programming:before{content:'r';background:#59ABE3}
```

Notice two mentions of `@media`…

goddamnyouryan commented:
@antstorm when i view the live css file i see two instances of media as well. I've gone ahead and pushed it again and invalidated again, clear your cache in 20 minutes and let me know if it has updated.

antstorm commented:
@goddamnyouryan right, I can see the changes now. I'm also on a different network… I wonder if proxy cache could have caused it. But anyways now it's fine. Thanks!
